killerbee/dev_zigduino.py

```diff
@@ -21,104 +21,103 @@
 
 class ZIGDUINO:
     def __init__(self, dev):
-	    '''
-	    Instantiates the KillerBee class for Zigduino running GoodFET firmware.
-	    @type dev:   String
-	    @param dev:  Serial device identifier (ex /dev/ttyUSB0)
-	    @return: None
-	    @rtype: None
-	    '''
-	    self._channel = None
-	    self._page = 0
-	    self.handle = None
-	    self.dev = dev
-	    self.handle = GoodFETatmel128rfa1()
-	    self.handle.serInit(port=self.dev)
-	    self.handle.setup()
-
-	    self.__stream_open = False
-	    self.capabilities = KBCapabilities()
-	    self.__set_capabilities()
+        '''
+        Instantiates the KillerBee class for Zigduino running GoodFET firmware.
+        @type dev:   String
+        @param dev:  Serial device identifier (ex /dev/ttyUSB0)
+        @return: None
+        @rtype: None
+        '''
+        self._channel = None
+        self.handle = None
+        self.dev = dev
+        self.handle = GoodFETatmel128rfa1()
+        self.handle.serInit(port=self.dev)
+        self.handle.setup()
+
+        self.__stream_open = False
+        self.capabilities = KBCapabilities()
+        self.__set_capabilities()
 
     def close(self):
-	    self.handle.serClose()
-	    self.handle = None
+	      self.handle.serClose()
+	      self.handle = None
 
     def check_capability(self, capab):
-    	return self.capabilities.check(capab)
+        return self.capabilities.check(capab)
 
     def get_capabilities(self):
-	    return self.capabilities.getlist()
+        return self.capabilities.getlist()
 
     def __set_capabilities(self):
-        '''
-        Sets the capability information appropriate for GoodFETAVR client and firmware.
-        @rtype: None
-        @return: None
-        '''
-        self.capabilities.setcapab(KBCapabilities.FREQ_2400, True)
-        self.capabilities.setcapab(KBCapabilities.SNIFF, True)
-        self.capabilities.setcapab(KBCapabilities.SETCHAN, True)
-        self.capabilities.setcapab(KBCapabilities.INJECT, True)
-        #self.capabilities.setcapab(KBCapabilities.PHYJAM_REFLEX, True)
-        self.capabilities.setcapab(KBCapabilities.SET_SYNC, True)
+          '''
```

Comment:
Check indents on this? `          ` shows 10 spaces, not a 4 multiple as would expect?